safe-global/safe-wallet-web
# Feat: create deep link page for messages

## What it solves

Resolves #2020

## How this PR fixes it
- Create a deep link page for displaying a single message.
- Link to this single message page from the sign message modal and from the message details in the history.

## How to test it
- Create a message for a 2/n safe (eg. log in to opensea via walletconnect)
- Sign the message using the first signer of the safe.
- Once the message has one signature, there should be a link to share the message with the other signers [1]
- Go to the message history. There should also be a share link in the message details, similar to transactions. [2]
- The link should lead to a page displaying that message by itself.

## Screenshots
[1]
![image](https://github.com/safe-global/safe-wallet-web/assets/17801424/6da4389e-8833-475a-994c-b5c49fa84986)
[2]
![image](https://github.com/safe-global/safe-wallet-web/assets/17801424/cf3d5f02-5db0-48d8-b70a-2ceabedfd994)
## Checklist
